modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -28,39 +28,44 @@
       />
       <AposButton
         type="primary" :label="saveLabel"
-        :disabled="docOtherFields.hasErrors || docUtilityFields.hasErrors"
+        :modifiers="buttonModifiers"
         @click="submit"
+        :tooltip="tooltip"
       />
     </template>
     <template #leftRail>
       <AposModalRail>
         <AposModalTabs
+          :key="tabKey"
           v-if="tabs.length > 0"
-          :current="currentTab" :tabs="tabs"
+          :current="currentTab"
+          :tabs="tabs"
+          :errors="fieldErrors"
           @select-tab="switchPane"
         />
       </AposModalRail>
     </template>
     <template #main>
       <AposModalBody>
         <template #bodyMain>
-          <AposModalTabsBody>
-            <div class="apos-doc-editor__body">
-              <AposSchema
-                v-if="docReady"
-                :schema="schemaOtherFields"
-                :current-fields="currentFields"
-                :trigger-validation="triggerValidation"
-                :utility-rail="false"
-                :following-values="followingValues('other')"
-                :doc-id="docId"
-                :value="docOtherFields"
-                @input="updateDocOtherFields"
-                :server-errors="serverErrors"
-                ref="otherSchema"
-              />
-            </div>
-          </AposModalTabsBody>
+          <div v-if="docReady" class="apos-doc-editor__body">
+            <AposSchema
+              v-for="tab in tabs"
+              v-show="tab.name === currentTab"
```

Comment:
Add all schemas instead of splitting the doc and doing a switcheroo when tabs change